sdk/protocol-manager-poc/src/implementation/aaveV3Plugin.ts

```diff
@@ -0,0 +1,167 @@
+import { AddressValue, Percentage, TokenAmount, TokenSymbol, Price, CurrencySymbol, RiskRatio } from "@summerfi/sdk-common/common"
+import type { AaveV3LendingPool, AaveV3PoolDebtConfig, AaveV3PoolCollateralConfig, AaveV3PoolId } from "@summerfi/sdk-common/protocols"
+import { PoolType, ProtocolName, EmodeType } from "@summerfi/sdk-common/protocols"
+import { Position, Token } from "@summerfi/sdk-common/common"
+import { BigNumber } from 'bignumber.js'
+import { z } from 'zod'
+import { ChainId, CreateProtocolPlugin, IPositionId, ProtocolManagerContext, ProtocolPlugin } from "../interfaces";
+import { AaveV3LikePluginBuilder, filterAssetsListByEMode } from "./aaveV3LikeBuilder";
+import { UNCAPPED_SUPPLY, PRESISION_BI } from "./constants";
```

Comment:
just wonedring... presision is that a thing or a typo?

Reply:
I believe it is a typo

Reply:
export const PRESISION = {
    WAD: 18,
    RAY: 27,
    RAD: 45,
}

export const PRESISION_BI = {
    WAD: 10n ** 18n,
    RAY: 10n ** 27n,
    RAD: 10n ** 45n,
}